.github/workflows/smoke-test-daily.yml

```diff
@@ -79,7 +79,7 @@ jobs:
         runs-on: ubuntu-20.04
         permissions:
             contents: read
-        needs: [api-tests]
+        #  needs: [api-tests]
```

Comment:
I decided to remove this for now so that the e2e tests run on daily regardless of the outcome of the API tests.